tests/test_wavex.py

```diff
@@ -0,0 +1,288 @@
+from io import StringIO
+import pytest
+import numpy as np
+import logging
```

Comment:
I don't think you need to use logging in the test.  You can just use `print()` statements, which are generally suppressed unless you run with `pytest -s`.  But if you want to use logging that's OK.

Reply:
If you are using logging, it's better to use loguru since that's what the rest of pint uses.